packages/alea-frontend/components/ProblemList.tsx

```diff
@@ -72,7 +75,9 @@ const ProblemList: FC<ProblemListProps> = ({ courseSections, courseId }) => {
 
   const seeSectionProblems = (sectionUri?: string, sectionTitle?: string) => {
     router.push(
-      `/per-section-quiz?sectionUri=${encodeURIComponent(sectionUri)}&courseId=${courseId}&sectionTitle=${encodeURIComponent(sectionTitle)}`
+      `/per-section-quiz?sectionUri=${encodeURIComponent(
```

Comment:
Unrelated to this PR but a better way to handle query params. I think we can also skip `encodeURIComponent` with this:

router.push({ pathname, query: { sectionUri: '...', sectionTitle: '...' } },